crates/sui-graphql-rpc/src/types/object.rs

```diff
@@ -702,6 +726,10 @@ impl Object {
         }
     }
 
+    pub(crate) fn root_version(&self) -> Option<u64> {
```

Comment:
nit: since this is what gets called elsewhere, I think we'll benefit from some doc comments explaining what we might return when calling `root_version()`.